app/src/test/java/com/codesoom/assignment/ToyServiceTest.java

```diff
@@ -0,0 +1,260 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.exceptions.ToyNotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("ToyService 클래스")
+public class ToyServiceTest {
+    private ToyService toyService;
+
+    @BeforeEach
+    void setUp() {
+        toyService = new ToyService();
+    }
+
+    private Toy randomToy() {
+        int randomNumber = (int) (Math.random() * 1000);
+        return new Toy(
+                "Toy" + randomNumber,
+                "Maker" + randomNumber,
+                randomNumber,
+                "URL" + randomNumber
+        );
+    }
+
+    @Nested
+    @DisplayName("getToys 메소드는")
+    class Describe_getToys {
+
+        @Nested
+        @DisplayName("아무런 toy가 등록되지 않았다면")
+        class Context_no_toys {
+
+            @Test
+            @DisplayName("빈 리스트를 리턴한다")
+            void it_returns_empty_list() {
+                List<Toy> toys = toyService.getToys();
```

Comment:
실험 결과를 확정한다는 의지를 보여주기 위해 이렇게 할 수도 있겠죠.
```suggestion
                final List<Toy> toys = toyService.getToys();
```

Reply:
적용했습니다 🔥